### DISCUSSION TITLE
New Torrust brand colors theme

### DISCUSSION THREAD

**Discussion by mario-nt** [torrust/torrust-index-gui#638] [Proposals]
I think it is worth considering the possibility of adding a new theme to the index GUI with the new branding colors, it is a really bold color but I think it kind of works, with a little adjustments to the saturation it should be less straining to the eye. Besides, we can keep the old orange as a "retro" or "classic" Torrust theme.

Some screenshots with quick mockups:

![image](https://github.com/user-attachments/assets/fc75ce3e-4684-4281-aad7-a6c795345c44)
![image](https://github.com/user-attachments/assets/05f72ed3-ee7e-417c-9404-6f980d4117fc)
![image](https://github.com/user-attachments/assets/c056b96d-383e-473c-a2fe-b5f360e1da57)
![image](https://github.com/user-attachments/assets/88a176ac-bdac-48f8-8a1b-3f426a1852d6)
![image](https://github.com/user-attachments/assets/028e2872-1671-4751-bf4a-2e0c7e74ce7c)
![image](https://github.com/user-attachments/assets/e42e0fb7-e8aa-4849-beae-7a2fbb7a881e)
![image](https://github.com/user-attachments/assets/7b3a2d57-d1c9-47dc-a964-2a53fd641d17)

**Comment by mario-nt** [comment]
@josecelano @cgbosse @da2ce7 @grmbyrn

**Reply by josecelano** [reply]
I think, it would be fine to have a similar look, but I don't think it's necessary to "force" the same style between the org website and this app style. A style could be convenient for a website but not for a webapp. If it's possible and applying the same style doesn't impact usability, I would do it, but I would not spend time on that. I think it would be more beneficial for users to customize the app. In fact, many users have requested that feature. Ultimately, end users don't care about Torrust style; they usually want to apply their own.
